mindjunkies/dashboard/tests/test_views.py

```
@@ -1,204 +1,191 @@
 import pytest
-from django.contrib.messages import get_messages
 from django.urls import reverse
 from model_bakery import baker
-from django.core.files.uploadedfile import SimpleUploadedFile
-from django.contrib.auth.models import Permission
-from django.contrib.contenttypes.models import ContentType
 
 from mindjunkies.accounts.models import User
 from mindjunkies.courses.models import Course, Enrollment
-from mindjunkies.dashboard.models import Certificate, TeacherVerification
+from mindjunkies.dashboard.models import TeacherVerification, Certificate
```

Ruff (0.8.2)

`mindjunkies.dashboard.models.Certificate` imported but unused

Remove unused import: `mindjunkies.dashboard.models.Certificate`

(F401)